mongodb/install.sls

```diff
@@ -0,0 +1,18 @@
+{% from "mongodb/map.jinja" import mongodb with context %}
+
+add_mongodb_package_repository:
+  pkgrepo.managed:
+    - name: {{ mongodb.repo }}
+    - humanname: MongoDB 2.6.5 Repository
+    - dist: {{ mongodb.repo_dist }}
+    - keyid: {{ mongodb.key }}
+    - keyserver: {{ mongodb.keyserver }}
+    - refresh_db: True
```

Comment:
You  should probably actually make this an optional step with a pillar value. Something like `{% if salt.pillar.get('add_pkgrepo') %}...{% endif %}`